README.md

```diff
@@ -32,7 +32,10 @@ and to your application module `build.gradle`:
 
 ```groovy
 dependencies {
-    compile "info.metadude.kotlin.library.c3media:c3media-base:${version}"
+    // use this for the basic c3media version
+    compile "info.metadude.kotlin.library.c3media:base:${version}"
```

Comment:
Please remove curly braces around the inline variable since they are actually not needed.